test/e2e/specs/active-links.js

```diff
@@ -71,6 +71,9 @@ module.exports = {
           .cssClassPresent(`li:nth-child(${i}) a`, 'router-link-exact-active')
           .assert.cssClassPresent(`li:nth-child(${i}) a`, 'router-link-active')
           .assert.attributeEquals(`li:nth-child(${i}) a`, 'aria-current', 'page')
+        if (i < 19) {
+          browser.assert.not.attributeEquals(`li:nth-child(${i + 1}) a`, 'aria-current', 'page')
```

Comment:
Let's remove this test and instead add the one I said above

Reply:
I'm not 100% sure what you were suggesting above, but I tried adding a test at line 33 instead. I had thought "line 33" above was a typo since the comment was on line 73 -- if this change isn't what you were thinking, I might need clarification :)

Reply:
Ah I see, sorry it wasn't clear. What you added is exactly what I had in mind!